backend/backend.py

```diff
@@ -179,7 +203,7 @@ def send_recording_status(self) -> None:
         self.prev_recording_status_time = time()
 
     def send_recording_error(self) -> None:
-        """ Send the most recent exception to the recording status MQTT topic. """
+        """Send the most recent exception to the recording status topic."""
```

Comment:
Why did the linter remove "MQTT"?

Reply:
Oh I removed it, AHAHHAHA. I had to make the line shorter than 79 characters.